pandas/core/indexing.py

```diff
@@ -640,14 +642,15 @@ def _get_setitem_indexer(self, key):
         Convert a potentially-label-based key into a positional indexer.
         """
         if self.name == "loc":
+            # always holds here bc iloc overrides _get_setitem_indexer
             self._ensure_listlike_indexer(key)
 
         if isinstance(key, tuple):
             for x in key:
                 check_deprecated_indexers(x)
 
         if self.axis is not None:
-            return self._convert_tuple(key)
```

Comment:
can we kill _convert_tuple?

Reply:
not sure yet; overarching goal is dealing with some trouble involving bool indexers